tests/test_replacer.py

```diff
@@ -401,6 +403,38 @@ def test_query_time_flags_groups(self) -> None:
             {ReplacementType.EXCLUDE_GROUPS},
         )
 
+    @mock.patch.object(settings, "REPLACER_MAX_GROUP_IDS_TO_EXCLUDE", 5)
+    def test_query_time_flags_bounded_size(self) -> None:
+        redis_client.flushdb()
+        project_id = 4
+        now = time.time()
```

Comment:
We have `freezegun` in the snuba repo you can use to set the time. Then you don't need to manually pass the time into the function (which I would like to avoid).

https://github.com/getsentry/snuba/blob/master/tests/test_optimize_scheduler.py#L249 <- Example